### PR TITLE
command: distinguish different states in the flashback tag statistics

### PR DESCRIPTION
**Pull request by JmPotato**
### What is changed and how it works?

Issue Number: ref #14788.

What's Changed:

```commit-message
- Distinguish different states in the flashback tag statistics.
- Move `incr_cmd_metric` into `schedule_command` to make sure all commands could be counted in.
```

### Check List 

Tests 

- Unit test
- Integration test

Once have the flashback run once:

image

image

### Release note 

```release-note
None.
```

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by JmPotato** [comment]
@overvenus PTAL, thx.

File: src/storage/txn/commands/flashback_to_version.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
-
```
Comment (HuSharp) [inline src/storage/txn/commands/flashback_to_version.rs:2]: It looks like the other files kept this line.

Comment (JmPotato) [reply src/storage/txn/commands/flashback_to_version.rs]: Fixed.

**Review by HuSharp — APPROVED** [review]
Can you show a picture of the result? reset LGTM!

**Review by overvenus — APPROVED** [review]

**Pull request merged**